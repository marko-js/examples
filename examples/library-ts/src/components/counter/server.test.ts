import { render, screen } from "@marko/testing-library";
import { composeStories } from "@storybook/marko";
import * as stories from "./stories";

for (const [name, story] of Object.entries(composeStories(stories))) {
  test(name, async () => {
    await render(story);
    expect(screen.getByRole("button").outerHTML).toMatchSnapshot();
  });
}
