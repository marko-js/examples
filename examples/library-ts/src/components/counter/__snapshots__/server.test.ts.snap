// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Default 1`] = `"<button>0</button>"`;

exports[`Double 1`] = `"<button>0</button>"`;
